public/media_engagement/ahadi_media_engagement.html

```diff
@@ -6,9 +6,14 @@
     <link rel="shortcut icon" type="image/png" href="../../public/coda/favicon.png" />
     <title>Engagement & Media Dashboard</title>
     <!-- Compiled and minified CSS -->
-    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/css/materialize.min.css">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```

Comment:
`crossorigin="anonymous"`
I've not seen this before. What do we need it for?

Reply:
Which library needs jquery?

Reply:
Bootstrap

Reply:
> `crossorigin="anonymous"`
> I've not seen this before. What do we need it for?
It's from the CDN but I will create another PR for the dependencies to resolve the issue 70

Reply:
thanks